Update replicated-sdk-overview.md (#1350)
replicatedhq/replicated-docs: docs/vendor/replicated-sdk-overview.md

You can run the Replicated SDK in integration mode to more quickly test new functionality for your application. Integration mode allows you to use mock data to test changes locally without having to create a release in the vendor portal and then install your Helm chart from the Replicated registry.

For more information, see [Developing Against the SDK API (Beta)](replicated-sdk-development).

> ## SDK-Enabled Helm Charts with KOTS

Reviewer: I could also see this being called "Limitations", but as we only have one limitation right now, I think this heading does make the info easier to find. I could see people wondering "how's this thing work with kots?"

You can run the Replicated SDK in integration mode to more quickly test new functionality for your application. Integration mode allows you to use mock data to test changes locally without having to create a release in the vendor portal and then install your Helm chart from the Replicated registry.

For more information, see [Developing Against the SDK API (Beta)](replicated-sdk-development).

## Using an SDK-enabled Chart in a KOTS Release

> Many vendors will find it convenient to use the same Helm Chart for KOTS releases that they use for Helm CLI releases, in some cases even exposing the same Replicated release to both sets of customers. However, running both KOTS and the SDK can cause duplication of instance data in the Vendor Portal and related APIs. To avoid this, you will need to create a single Helm Chart that contains the Replicated SDK, but with a conditional value to exclude the SDK when the chart is being deployed by KOTS. To do this, you will need to make three changes to your application.

Reviewer 2: Why "single helm chart". I think this could be more accurately described as:

To avoid this, any of your Helm Charts that contain the Replicated SDK as a depedency or subchart will need to contain a conditional value to exclude the SDK...

> Many vendors will find it convenient to use the same Helm Chart for KOTS releases that they use for Helm CLI releases, in some cases even exposing the same Replicated release to both sets of customers. However, running both KOTS and the SDK can cause duplication of instance data in the Vendor Portal and related APIs. To avoid this, you will need to create a single Helm Chart that contains the Replicated SDK, but with a conditional value to exclude the SDK when the chart is being deployed by KOTS. To do this, you will need to make three changes to your application.

Author: ```suggestion
Many vendors will find it convenient to use the same Helm Chart for KOTS releases that they use for Helm CLI releases, in some cases even exposing the same Replicated release to both sets of customers. However, running both KOTS and the SDK can cause duplication of instance data in the Vendor Portal and related APIs. To avoid this, any of your Helm Charts that contain the Replicated SDK will need a conditional value to exclude the SDK when the chart is being deployed by KOTS. To do this, you will need to make three changes to your application.
```
